templates/Includes/UserForm.ss

```diff
@@ -9,13 +9,16 @@
 	<p id="{$FormName}_error" class="message $MessageType" aria-hidden="true" style="display: none;"></p>
 <% end_if %>
 
-<fieldset>
-	<% if $Legend %><legend>$Legend</legend><% end_if %>
-	<% loop $Fields %>
-		$FieldHolder
-	<% end_loop %>
-	<div class="clear"><!-- --></div>
-</fieldset>
+<% if $Legend %>
```

Comment:
Decided to check for fieldset as a module can in fact override UserForm and add a legend, this way not dropping support for them.